source/binder.cpp

```diff
@@ -13,6 +13,8 @@
 #include <binder.hpp>
 
 // Declares clang::SyntaxOnlyAction.
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/ASTContext.h"
```

Comment:
Could you please double check if these includes should not be need here? Or is it due to high version of LLVM  changes?

Reply:
👍